vochain/offchaindatahandler/census.go

```diff
@@ -30,30 +26,3 @@ func (d *OffChainDataHandler) enqueueOffchainCensus(root, uri string) {
 	}
 	d.storage.AddToQueue(uri, d.importExternalCensus, true)
 }
-
-// importRollingCensus imports a rolling census (zkIndexed) from a remote URI into the censusDB storage.
-func (d *OffChainDataHandler) importRollingCensus(pid []byte) {
```

Comment:
I'd not remove this method, we will need it for supporting rolling-census voting. Ideally this should be adapted to work without indexed.

Reply:
I will restore the changes regarding to rolling census and open an issue to adapt it, ok?